src/pages/vault/docs/[[...page]].tsx

```diff
@@ -0,0 +1,34 @@
+import { ReactElement } from 'react'
+import vaultData from 'data/vault.json'
+import { Product } from 'types/products'
+import { getStaticGenerationFunctions } from 'layouts/docs/server'
+import DocsLayout from 'layouts/docs'
+import DocsView from 'views/docs-view'
+import Columns from 'components/author-primitives/vault/columns'
+import InlineTag from 'components/author-primitives/vault/inline-tag'
+import { vaultUrlAdjuster } from 'layouts/docs/utils/product-url-adjusters'
+
+const basePath = 'docs'
+const baseName = 'Docs'
+const product = vaultData as Product
+const additionalComponents = {
```

Comment:
Our `docs/server` utils now include a base of shared MDX components only. This allows us to add custom components per-route, to match existing configuration (and to avoid conflicts & page bloat where custom components are not used).

For example, `/vault/docs` requires additional custom components, but `/vault/api-docs` does not.